Source/WebInspectorUI/UserInterface/Views/TreeOutline.js

```diff
@@ -363,9 +363,9 @@ WI.TreeOutline = class TreeOutline extends WI.Object
 
     removeChildren(suppressOnDeselect)
     {
-        for (let child of this.children) {
-            child.deselect(suppressOnDeselect);
+        this.selectedTreeElement?.deselect(suppressOnDeselect);
```

Comment:
it's possible for more than one thing to be selected, so i think this actually should be
```js
for (let selectedTreeElement  of this.selectedTreeElements)
    selectedTreeElement.deselect(suppressOnDeselect);
```